### PR TITLE
make facebook albums full width

### PR DESCRIPTION
**Pull request by nvpmai95**
image

### CODE REVIEW

File: app/views/news_items/galleries.html.erb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div class="fb-album"></div>
 
 <script type="text/javascript">
-  $(document).ready(function() {
+  $(document).ready(function () {
```
Comment (harley) [inline app/views/news_items/galleries.html.erb]: there shouldnt be a space here, should be `function() {`

Comment (harley) [inline app/views/news_items/galleries.html.erb]: also it should be `document.on("turbolinks:load"`, not `document.ready`

Comment (nvpmai95) [inline app/views/news_items/galleries.html.erb]: If i use 
`$(document).on("turbolinks:load", function() {...` the albums can only be loaded on the first time I go to that page. When i go to other pages and come back, it becomes really slow and the albums loading runs like forever

**Comment by nvpmai95** [comment]
I'll merge this branch. Basically, it's done with the task: make facebook albums display in full width. I've noted down the problem in Obstacles. We'll solve this in another branch

**Pull request merged**